src/components/tables/TableConnected.tsx

```diff
@@ -57,14 +59,41 @@ export const getTableCompositeState = (state: IReactVaporState, id: string): ITa
     } as ITableCompositeState;
 };
 
-const mapStateToProps = (state: IReactVaporState, ownProps: ITableOwnProps): ITableCompositeStateProps => ({
-    tableCompositeState: getTableCompositeState(state, ownProps.id),
-});
+const getDataById = (data: ITableData, props: ITableOwnProps): ITableById => data.byId;
+
+const getSelectedIds = (data: ITableData, props: ITableOwnProps): string[] => data.selectedIds || [];
+
+const getActions = (data: ITableData, props: ITableOwnProps): (rowData?: IData) => IActionOptions[] => props.getActions;
+
+const getMultiSelect = (data: ITableData, props: ITableOwnProps): boolean => props.rowsMultiSelect;
+
+const actionsSelector = createSelector([getDataById, getSelectedIds, getMultiSelect, getActions], (byId: ITableById, selectedIds: string[], isMultiSelect: boolean, getAction: (rowData?: IData) => IActionOptions[]): IActionOptions[] => {

```

Comment:
you should probably put each argument on its own line here, the line seems very long

Reply:
I tried but the lintfix doesnt understand this line and it add an error.